app/assets/javascripts/application.js

```diff
@@ -38,4 +38,15 @@
 
 // =============================
 
-$(function(){ $(document).foundation() });
\ No newline at end of file
+$(function(){
+  $(document).foundation({
+  "magellan-expedition": {
```

Comment:
Identifier 'active_class' is not in camel case.<br>Mixed double and single quotes.